src/index.html

```diff
@@ -22,6 +22,404 @@
+            <p class="card__text">Price:</p>
+            <p class="card__amount">$2,199</p>
+          </div>
+
+          <a
+            href="#"
+            class="card__button"
+          >
+            buy
+          </a>
+        </div>
+      </div>
+      <div class="card">
+        <div class="card__img">
+          <img
+            src="./images/imac.jpeg"
+            alt="imac_logo"
+          />
+        </div>
+        <div class="card__container">
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code card-text">Product code: 195434</p>
+
+          <div class="card__rating">
+            <ul class="card__stars stars stars--4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+
+            <p class="card__reviews card-text">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__text">Price:</p>
+            <p class="card__amount">$2,199</p>
+          </div>
+
+          <a
+            href="#"
+            class="card__button"
+          >
+            buy
+          </a>
+        </div>
+      </div>
+      <div class="card">
+        <div class="card__img">
+          <img
+            src="./images/imac.jpeg"
+            alt="imac_logo"
+          />
+        </div>
+        <div class="card__container">
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code card-text">Product code: 195434</p>
+
+          <div class="card__rating">
+            <ul class="card__stars stars stars--4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+
+            <p class="card__reviews card-text">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__text">Price:</p>
+            <p class="card__amount">$2,199</p>
+          </div>
+
+          <a
+            href="#"
+            class="card__button"
+          >
+            buy
+          </a>
+        </div>
+      </div>
+      <div class="card">
+        <div class="card__img">
+          <img
+            src="./images/imac.jpeg"
+            alt="imac_logo"
+          />
+        </div>
+        <div class="card__container">
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code card-text">Product code: 195434</p>
+
+          <div class="card__rating">
+            <ul class="card__stars stars stars--4">
```

Comment:
```suggestion
            <ul class="stars stars--4">
```
Do you need card_stars class, let stars block be independent so that we follow BEM